Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -69,6 +71,15 @@ - (instancetype)initWithOptions:(SentryOptions *)options
 
 - (void)start
 {
+    SentryBinaryImageCache *imageCache = SentryDependencyContainer.sharedInstance.binaryImageCache;
+
+    for (NSString *string in self.inAppLogic.inAppIncludes) {
```

Comment:
`h`: Just double-checking why this works now: Previously, we only looked for UIViewControllers in the image of the app delegate class. Now we use inAppIncludes which per default includes the `CFBundleExecutable` which is basically the same image of the app delegate class. Could it happen that the UIApplication returned by `findApp` is not in the `CFBundleExecutable`? If yes, this would be a breaking change and we still should call `swizzleAllSubViewControllersInApp` after swizzling the inApp ones. There is no risk of double swizzling because if we already swizzled the UIViewController again swizzling it does nothing if I'm not mistaken.

Reply:
@brustolin, please answer the question above ⬆️.